src/porepy/grids/gmsh/mesh_2_grid.py

```diff
@@ -113,10 +113,34 @@ def create_2d_grids(pts, cells, **kwargs):
             g_2d.append(g)
 
     else:
+
         triangles = cells["triangle"].transpose()
         # Construct grid
         g_2d = simplex.TriangleGrid(pts.transpose(), triangles)
 
+        # save all the extra tags for the grids
```

Comment:
This is a place where I miss comments on two levels. First, what is this new piece of code doing, on a high level. Second, what is the role of the second for loop.

Reply:
OK, done.